client/signup/config/flows-pure.js

```diff
@@ -90,6 +92,14 @@ export function generateFlows( {
 			providesDependenciesInQuery: [ 'toStepper' ],
 			optionalDependenciesInQuery: [ 'toStepper' ],
 			hideProgressIndicator: true,
+			postCompleteCallback: async ( { dispatch } ) => {
+				dispatch(
+					savePreference( READER_AS_LANDING_PAGE_PREFERENCE, {
+						useReaderAsLandingPage: true,
+						updatedAt: Date.now(),
+					} )
+				);
+			},
```

Comment:
I'm not sure this is the best place for this, but I'm struggling to find another place to put this after the user has been created.